File: src/main/java/com/example/khuasenov/fluxstarter/model/Persons.java
package com.example.khuasenov.fluxstarter.model;

import java.util.List;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

/**
 * @author Khusaenov on 17.07.2018
 */
@Document
@Getter
@Setter
@EqualsAndHashCode
//@NoArgsConstructor
//@AllArgsConstructor
public class Persons {

    @Id
    private String id;

    private String name;

    private String surname;

    private Integer age;

    private List<String> options;


}

File: src/main/kotlin/com/example/khusaenov/fluxstarter/model/Persons.kt
package com.example.khusaenov.fluxstarter.model

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

/**
 * @author Khusaenov on 10.08.2018
 */
@Document(collection = "persons")
data class Persons(@Id val id: String, val name: String?, val surname: String?, val age: Int?, val options: List<String>?) {
}